caikit/runtime/http_server.py

```diff
@@ -106,15 +106,18 @@ class RuntimeHTTPServer(RuntimeServerBase):
     ## Interface ##
     ###############
 
-    def __init__(self, tls_config_override: Optional[aconfig.Config] = None):
+    def __init__(
+        self,
+        inference_service: ServicePackage,
+        #  # pylint: disable=unused-argument
+        training_service: Optional[ServicePackage],
+        tls_config_override: Optional[aconfig.Config] = None,
+    ):
         super().__init__(get_config().runtime.http.port, tls_config_override)
 
         self.app = FastAPI()
 
         # Set up the central predict servicer
-        inference_service = ServicePackageFactory().get_service_package(
```

Comment:
What are the `Optional` semantics on these args supposed to mean? I could see the answer being either "use the global singleton" or "run without this service"

Reply:
I assumed they meant that we _can_ start without a `trainer_service` - which makes sense since we might not always have training